pkgs/applications/networking/instant-messengers/pidgin-plugins/purple-slack/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, pidgin }:
+
+stdenv.mkDerivation {
+  pname = "purple-slack";
```

Comment:
```suggestion
  pname = "purple-slack-unstable";
```

Reply:
See: https://github.com/NixOS/nixpkgs/issues/68531

Reply:
Should be resolved in latest force push :)